### PR TITLE
Add link to bob-barker

### PR DESCRIPTION
**Pull request by esjr81**

### CODE REVIEW

**Comment by esjr81** [comment]
Please review this, @casey-collab.

File: english/breakfast/breakfast.md [hunk 1]
```diff
@@ -11,3 +11,5 @@ have to go to the supermarket. That same supermarket that you vowed you would ne
 
 But, the wall has to be fixed.  So you gather your courage and go to the dreaded supermarket and make your way to the
 aisle with the marshmallows.  Turning into the aisle you see a terrible sight.
+
+[Who is this man?](english/bob-barker/barker.md)
```
Comment (casey-collab) [inline english/breakfast/breakfast.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by esjr81** [comment]
Please review this, @casey-collab.

**Comment by casey-collab** [comment]
Looks great! :penguin:

**Pull request merged**